tests/robot-tests/tests/general_public/redirects.robot

```diff
@@ -7,29 +7,39 @@ Test Setup          fail test fast if required
 
 Force Tags          GeneralPublic    Local    Dev    Test    Preprod
 
+*** Variables ***
+${PUBLIC_URL}=    %{PUBLIC_URL}
+${PUBLIC_URL_WITHOUT_AUTH}           ${EMPTY}
+
+#In local, public url contains authentication details. In order to make it work while running against dev- we need to get rid of it to compare public url against expected value.
 
 *** Test Cases ***
+
+Parse and Store public url without auth details
+    ${PUBLIC_URL_WITHOUT_AUTH}=    remove auth from url    %{PUBLIC_URL}
+    set suite variable    ${PUBLIC_URL_WITHOUT_AUTH}
+
 Verify that absolute paths with trailing slashes are redirected without them
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains   ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
 
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains   ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
 
     user navigates to public frontend    %{PUBLIC_URL}/glossary/?someRandomUrlParameter=123
     user waits until page contains    Glossary
-    user checks url equals    %{PUBLIC_URL}/glossary?someRandomUrlParameter=123
+    user checks removed auth from url contains    ${PUBLIC_URL_WITHOUT_AUTH}/glossary?someRandomUrlParameter=123
 
     # Would be amazing if we could verify these redirects were done with a 301 rather than a 308...
 
 Verify that redirects do not affect browser history
     user navigates to    about:blank
 
     user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/
-    user waits until page contains    Browse our open data
-    user checks url equals    %{PUBLIC_URL}/data-catalogue
+    user waits until page contains    Data catalogue
+    user checks removed auth from url contains   ${PUBLIC_URL_WITHOUT_AUTH}/data-catalogue
```

Comment:
A suggestion to edit the wording of the keyword like so
'user checks url without auth equals'